tasks/10_remove-whitespaces/remove-whitespaces.js

```diff
@@ -0,0 +1,3 @@
+export function trimWhiteSpaces(str) {
+  return str.replace(/ /g, '');
```

Comment:
Why `/ /g` and not `' '`?

Reply:
When passing a string, instead of a regex expression as the first parameter,  it only replaces the first occurrence of the string and  therefore doesn't work for all cases and fails the unit tests.